tests/conversion/advanced-converters.js

```diff
@@ -40,7 +42,8 @@ describe( 'advanced-converters', () => {
 
 		const editing = new EditingController( model );
 
-		viewRoot = editing.createRoot( 'div' );
+		editing.view.attachDomRoot( document.createElement( 'div' ) );
+		viewRoot = editing.view.getRoot();
```

Comment:
Can't you just set the name of the view root element without attaching it to the DOM (with a comment what is going on)? Tests which do not use DOM are usually more stable.